### PR TITLE
{bio,vis}[intel/2019b] deepTools v3.3.1, plotly.py v4.4.1 w/ Python 2.7.16 + 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#9479~~ (missing wcwidth extension in Python 2.7.16)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/bf18e13fcb3278a8e24bfcddd7417e20 for a full test report.

**Comment by verdurin** [comment]
@boegel any chance of a test report?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/85098c9ad730a69a45cb5218c16babdd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2411.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ac077caf0b22b01f3f6b606ea61fe9c3 for a full test report.

File: easybuild/easyconfigs/p/plotly.py/plotly.py-4.4.1-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('retrying', '1.3.3', {
+        'checksums': ['08c039560a6da2fe4f2c426d0766e284d3b736e355f8dd24b37367b0bb41973b'],
+    }),
+    ('plotly', version, {
```
Comment (verdurin) [inline easybuild/easyconfigs/p/plotly.py/plotly.py-4.4.1-intel-2019b.eb]: @boegel for consistency, use `name` here?

**Comment by verdurin** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/p/plotly.py/plotly.py-4.4.1-intel-2019b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/plotly.py/plotly.py-4.4.1-intel-2019b.eb]: Changing that was a mistake, see #9599...

**Pull request merged**